http/proxy/server.go

```diff
@@ -162,22 +179,22 @@ func (server *Server) verifyAccess(ctx echo.Context, request *http.Request, toke
 	// §6.2.2 other resources that require a credential and a user contract
 	// the existence of the user contract is validated by validateWithNutsAuthorizationCredential
 	if err := server.validateWithNutsAuthorizationCredential(request.Context(), token, *route); err != nil {
-		fmt.Errorf("access denied for %s on %s: %w", route.operation, route.path(), err)
+		logrus.Errorf("access denied for %s on %s: %#v", route.operation, route.path(), err)
```

Comment:
what does the `#` do?

Reply:
The `%#v` basically means: "Give me a Go representation of this value".